packages/@react-aria/checkbox/docs/useCheckboxGroup.mdx

```diff
@@ -196,6 +198,31 @@ function Example() {
 }
 ```
 
+### Description
```

Comment:
Doesn't look like the description/error message for these checkbox group/radio group aria docs are being properly associated with the group itself (no aria-describedby).

Reply:
I don't see this when I look at the docs for TextField #help text.